opendj-packages/pom.xml

```diff
@@ -117,10 +113,8 @@
 
                 <!--
                   !  Prevent useless jar files generation. Needed for:
-                  !    * opendj-rpm-standard
-                  !    * opendj-rpm-oem
-                  !    * opendj-deb-standard
-                  !    * opendj-deb-oem
+                  !    * opendj-rpm
+                  !    * opendj-deb
                  -->
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
```

Comment:
AFAIK this is useless as the packaging is of type POM. Default JAR goal gets configured only for JAR packaging.

I have noticed that the former packages were probably of type JAR. But POM is better for those submodules..